zhaquirks/tuya/ts0041.py

```diff
@@ -103,3 +103,80 @@ class TuyaSmartRemote0041TI(CustomDevice):
         (LONG_PRESS, BUTTON_1): {ENDPOINT_ID: 1, COMMAND: LONG_PRESS},
         (DOUBLE_PRESS, BUTTON_1): {ENDPOINT_ID: 1, COMMAND: DOUBLE_PRESS},
     }
+    
+
+class TuyaSmartRemote0041TOPlusA(CustomDevice, Tuya4ButtonTriggers):
```

Comment:
Remove `Tuya4ButtonTriggers` here

Reply:
I definitely don't know much about Tuya, but these remotes all look like they do not need the Tuya spell, so I don't see the benefit (and also not of applying it to just one remote model).